pa11ycrawler/spiders/edx.py

```diff
@@ -95,17 +127,63 @@ def start_requests(self):
                 )
             )
             # make sure to request a parseable JSON response
+            headers = {
+                b"Accept": b"application/json",
+            }
             yield scrapy.Request(
                 auth_url,
-                headers={b"Accept": b"application/json"},
-                callback=self.parse_auto_auth,
+                headers=headers,
+                callback=self.after_auto_auth,
             )
 
-    def parse_auto_auth(self, response):
+    def after_initial_csrf(self, response):
```

Comment:
I'm not sure what errors it might encounter. If you can give me some suggestions, I'd be happy to add error handling for those cases, and write tests to cover them as well.